<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>702</width>
    <height>497</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SmartCar2016</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_bmp">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>320</width>
      <height>240</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>beijing.bmp</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_start">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>430</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>开始</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_stop">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>430</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>停止</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_logout">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>30</y>
      <width>320</width>
      <height>240</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>10</y>
      <width>341</width>
      <height>271</height>
     </rect>
    </property>
    <property name="title">
     <string>输出信息</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>341</width>
      <height>271</height>
     </rect>
    </property>
    <property name="title">
     <string>图像输出</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_com">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>430</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_px">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>430</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <zorder>groupBox</zorder>
   <zorder>label_bmp</zorder>
   <zorder>pushButton_start</zorder>
   <zorder>pushButton_stop</zorder>
   <zorder>textBrowser_logout</zorder>
   <zorder>groupBox_2</zorder>
   <zorder>comboBox_com</zorder>
   <zorder>comboBox_px</zorder>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
